controller/src/beerocks/bml/bml.cpp

```diff
@@ -686,7 +686,7 @@ int bml_get_dcs_scan_results(BML_CTX ctx, const char *radio_mac,
                              unsigned int *output_results_size, unsigned char *output_result_status,
                              bool is_single_scan)
 {
-    // Validate intput params;
+    // Validate input parameters
```

Comment:
usually commits that changes existing code should come first (will also make it easier for you to rebase on PR change requests)
nitpick

Reply:
Moved